src/applications/static-pages/i18Select/I18Select.js

```diff
@@ -21,24 +20,17 @@ const I18Select = ({ baseUrls, languageCode }) => {
           }
           return (
             <span key={i}>
-              <a
-                className={`vads-u-font-size--base vads-u-font-family--sans vads-u-padding-bottom-0p5 ${
+              <va-link
+                className={`i18-select vads-u-font-family--sans vads-u-padding-bottom-0p5 ${
                   languageConfig.code === languageCode
                     ? 'vads-u-font-weight--bold vads-u-color--base vads-u-text-decoration--none'
                     : ''
                 }`}
-                onClick={_ => {
-                  recordEvent({
-                    event: 'nav-pipe-delimited-list-click',
-                    'pipe-delimited-list-header': languageConfig.code,
-                  });
-                }}
                 href={baseUrls[languageConfig.code]}
                 hrefLang={languageConfig.code}
                 lang={languageConfig.code}
-              >
-                {languageConfig.label}{' '}
-              </a>
+                text={`${languageConfig.label} ${' '}`}
```

Comment:
I don't think the `${' '}` is needed in the `text` prop. It was previously used to create space between the link text and the pipe. If it's still needed, it can be added after the `<va-link />`

Reply:
@laflannery Question here on the current page for the language toggle. Previously we added a text-decoration: none on the `<a>` for this toggle. But in va-link we cannot apply this style. Spoke with Randi about making the current page a regular `<p>` instead of a va-link, any accessibility concerns there?

Reply:
I actually would want this to follow a pattern more closely aligned with the other navigational elements, like breadcrumbs, however this is going to be [made into an official component](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2583) by the DST.

I think removing the link on the active page would be the best option given that `<va-link>` doesn't have an `aria-current` attribute available